File: Selenium/Activities/Activity2.py
import selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

#from selenium import By
with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/login-form")
    print("page title is "+driver.title)
    username=driver.find_element(By.ID,"username").send_keys("admin")
    password=driver.find_element(By.ID,"password").send_keys("password")
    #driver.find_element(By.XPATH,"//button[text(),'Log in']").click()
    login = driver.find_element(By.CSS_SELECTOR, "button.button:nth-child(4)")
    login.click()
    message=driver.find_element(By.ID,"action-confirmation").text
    print("Confirmation msg is :"+message)

File: Selenium/Activities/Activity2.java
import org.openqa.selenium.By;
import org.openqa.selenium.JavascriptExecutor;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity2 {
    public static void main(String[] args) {
        WebDriver driver=new ChromeDriver();
        driver.manage().window().maximize();
        driver.get("https://v1.training-support.net/selenium/login-form");
        System.out.println("The page title is  :"+driver.getTitle());
        // identifying username and entering user name
        driver.findElement(By.id("username")).sendKeys("admin");
        // identifying password and entering Password
        driver.findElement(By.id("password")).sendKeys("password");
        //Clicking on Sign in btn
        driver.findElement(By.xpath("//button[text()='Log in']")).click();
        String message=driver.findElement(By.id("action-confirmation")).getText();
        System.out.println("Log in confirmation message is :"+message);
        driver.close();
    }
}
